templates/frontend/objects/article_details.tpl

```diff
@@ -241,15 +262,32 @@
 				{/if}
 
 				{* Author biographies *}
-				{foreach from=$article->getAuthors() item=author key=authorKey}
+				{assign var="hasBiographies" value=0}
+				{foreach from=$article->getAuthors() item=author}
 					{if $author->getLocalizedBiography()}
-						<section>
-							<h2>{$author->getFullName()|escape}</h2>
-							<p>{$author->getLocalizedBiography()|strip_unsafe_html}</p>
-						</section>
+						{assign var="hasBiographies" value=$hasBiographies+1}
```

Comment:
The same here as in the previous comment, we can just break a loop after this assignment  - after the first author with biography is found and `hasBiographies` variable is updated.